oct/tournament/serializers.py

```diff
@@ -34,9 +34,9 @@ def _get_serializer_of_model(self, model) -> Type['Serializer']:
     def _transform(self, obj, fields, exclude, include):
         data = {}
         for field in fields:
-            field_type = getattr(self.model, field)
+            field_type = getattr(self.model, field, None)
             json_name = self.transforms[field] if field in self.transforms else field
-            value = getattr(obj, field)
+            value = getattr(obj, field, None)
```

Comment:
If the model or object is missing the field an error should probably be thrown

Reply:
the none is so the @property fields in the models could be put in the fields to be serialized